feature_layers/create-mobile-geodatabase/src/main/resources/create_mobile_geodatabase/main.fxml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2022 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.layout.Background?>
+<?import javafx.scene.layout.BackgroundFill?>
+<?import javafx.scene.paint.Color?>
+
+<?import com.esri.arcgisruntime.mapping.view.MapView?>
+
+<StackPane xmlns="http://javafx.com/javafx"
+           xmlns:fx="http://javafx.com/fxml"
+           fx:controller="com.esri.samples.create_mobile_geodatabase.CreateMobileGeodatabaseController"
+           stylesheets="/create_mobile_geodatabase/style.css">
+    <!--SDK MapView-->
+    <MapView fx:id="mapView"/>
+    <!--Controls box-->
+    <VBox StackPane.alignment="TOP_LEFT" spacing="5" maxWidth="180" maxHeight="100" styleClass="panel-region">
+        <padding>
+            <Insets topRightBottomLeft="10"/>
+        </padding>
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.7"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
+        <StackPane.margin><Insets left="10" top="10"/></StackPane.margin>
+        <Label  fx:id="label"
+                text="Create mobile geodatabase"/>
+        <Button fx:id="createGeodatabaseButton"
+                text="Create geodatabase"
+                textAlignment="CENTER"
+                prefWidth="150"
+                onAction="#createGeodatabase"/>
+        <Button fx:id="closeGeodatabaseButton"
+                text="Close geodatabase"
+                textAlignment="CENTER"
+                prefWidth="150"
+                disable="true"
+                onAction="#closeGeodatabase"/>
+        <Button fx:id="viewTableButton"
```

Comment:
let's swap the order of these two, with the work flow in mind:
1. User creates geodatabase
2. User adds features and views them in table
3. Finally user closes database

Reply:
Another thought here with the above workflow: should we make it so that a user is only able to close the geodatabase once they've viewed the table (i.e. when viewTableButton is clicked, set disable to false on the closeGeodatabaseButton)? I don't have strong opinions either way: Just a thought to help guide them via UI as they use the sample. Will leave this to your discretion!